layouts/shortcodes/slideshare.html

```diff
@@ -0,0 +1,7 @@
+<iframe
+  style='width: 100%; height: 500px;'
+  frameborder='0' marginwidth='0' marginheight='0' scrolling='no'
+  src='http://www.slideshare.net/slideshow/embed_code/{{.Get "id"}}?rel=0'
```

Comment:
You are passing unnamed parameter. But reading named parameter "id". It won't work.

Reply:
Done, Changed to `https`

Reply:
@MunifTanjim Already done respective changes to use named parameter.